### ISSUE TITLE
Don't make labels too descriptive

### ISSUE THREAD

**Issue by oliverjam** [fac25/week2-markup-Suraj-Yassien#23]
https://github.com/fac25/week2-markup-Suraj-Yassien/blob/5ff7f72c06eee791d56d5b7284d047859ff54656/index.html#L64-L68

It's awesome to see you using `aria-label` to provide descriptive labels for users who can't see these icons. You don't need to be _too_ descriptive though—since this is an `<a>` tag the screen reader will already tell the user what to do (e.g. something like "anchor tag, 'Click here to go to Suraj's Github Profile', hit return to navigate".

Also bear in mind a lot of screen reader users won't be able to "click here" since they can't see the link properly (they will probably be navigating with their keyboard).

You generally want to avoid descriptions that are part of the HTML element (stuff like "image of an x") or instructions that should be self-evident based on the element ("click here to x"). In this case a clear label would be something like "Suraj's GitHub profile".

**Comment by oliverjam** [comment]
Oh and don't forget the `aria-hidden` on your icons. You have it on most of them, it's just missing here
